code/_onclick/hud/screen_objects.dm

```diff
@@ -465,6 +467,17 @@
 
 	return TRUE
 
+/atom/movable/screen/storage/corner
+	name = "storage"
+	icon_state = "storage_corner_topleft"
+	screen_loc = "7,7 to 10,8"
+
+/atom/movable/screen/storage/rowjoin
+	name = "storage"
+	icon_state = "storage_rowjoin_left"
+	screen_loc = "7,7 to 7,7"
```

Comment:
shouldn't these screen locs be defines

Reply:
or is that not standard

Reply:
I should just axe these, not sure why they were there at all since its always set before displaying.